utah-design-system-website/src/react/components/websiteContent/library/components/containers/Card/CardDocumentation.jsx

```diff
@@ -48,10 +122,90 @@ export function CardDocumentation() {
           </ul>
         )}
       />
+      <PreCodeForCodeString
+        className="gray-block mt-spacing"
+        codeRaw={`
+          <div class="action-card action-card--primary-color action-card--solid">
+            <div class="action-card__title">
+              <h3>Getting Started</h3>
+              <span class="button--icon button--icon-right">
+                <span 
+                  class="utds-icon-before-arrow-right"
+                  aria-hidden="true"
+                  /></span>
+              </span>
+            </div>
+            <div class="action-card__body">
+              Learn how to implement the Design System.
+            </div>
+          </div>
+        `}
+      />
 
       <StaticExample
         title="Vertical Cards"
-        renderedExample={<LightBox image={verticalCardsScreenshot} alt="Vertical Cards" className="flex-3up-gap" />}
+        renderedExample={(
+          <div className="flex flex-col gap items-center">
+            <div className="card">
+              <img src={delicateArch} alt="Delicate Arch" className="card__image" />
+              <div className=" p-spacing-xl">
+                <div className="card__title">The Mighty 5</div>
+                <p>Sed ut perspiciatis unde omnis iste natus error sit
+                  voluptatem accusantium doloremque laudantium,
+                  totam rem aperiam, eaque ipsa quae ab illo inventore
+                  veritatis et quasi architecto beatae vitae dicta sunt
+                  explicabo voluptatem accusantium doloremque.
+                </p>
+                <div className="flex justify-end">
+                  <button type="button" className="button">Contact Us</button>
+                  <button type="button" className="button button--solid ml-spacing">
+                    See More
+                    <span className="button--icon button--icon-right"><span className="utds-icon-before-arrow-right" aria-hidden="true" /></span>
+                  </button>
+                </div>
+              </div>
+            </div>
+            <div className="card gray-color-background white-color">
+              <div>
+                <img src={nightPark} alt="Delicate Arch" className="card__image" />
+              </div>
+              <div className=" p-spacing-xl">
+                <div className="card__title">Dark Sky Parks in Utah</div>
+                <p>Sed ut perspiciatis unde omnis iste natus error sit
+                  voluptatem accusantium doloremque laudantium,
+                  totam rem aperiam, eaque ipsa quae ab illo inventore
+                  veritatis et quasi architecto beatae vitae dicta sunt
+                  explicabo voluptatem accusantium doloremque.
+                </p>
+                <div className="flex justify-end">
+                  <button type="button" className="button">Contact Us</button>
```

Comment:
These buttons have the wrong background color when hovered? they disappear into the background.

Reply:
Will fix